assets/src/components/account/AccountOverview.tsx

```diff
@@ -288,6 +324,14 @@ class AccountOverview extends React.Component<Props, State> {
             onSave={this.handleUpdate}
           />
         </Box>
+
+        <Checkbox
+          checked={hideWidgetOutsideWorkingHours}
+          onChange={this.handleUpdateHideWidgetOutsideWorkingHours}
+          disabled={isRefreshing}
+        >
+          Hide chat window outside of working hours
+        </Checkbox>
```

Comment:
Any preference on where the button lives? North/south of the Edit button? Went with quite literal text as well

Reply:
i think it's fine to have it here! though i'd maybe remove the word "window", i think it still makes enough sense without it... what do you think?

Reply:
makes sense to me! I like the elegant naming of "hideOutsideWorkingHours" as well

Reply:
done!